solutions/bronze/usaco-1251.mdx

```diff
@@ -0,0 +1,84 @@
+
+## Explanation
+
+Basically, the solution is using sorting and for loop to find the max tuition. I will explain it one by one.
+
+First, the type should be assigned as ```long long``` since the ```max tuition``` and the ```best tuition``` on that specific point can exceed the range of -2,147,483,648 to 2,147,483,647. The number of cows can be 1 to 100,000, while the ```maximum tuition``` varies from 1 to 1,000,000.
+
+Multiplying both numbers exceeds the range, so the array and variables that will hold the ```maxTuition``` and ```bestTuition``` on that specific point should be assigned with ```long long``` rather than ```int``` form.
+
+After assigning the variables, the ```number of cows``` and the ```maximum tuition``` that each cow is willing to pay should be plugged in.
+
+Next, the array that contains the ```maximum tuition``` that each cow is willing to pay should be sorted. The reason is to find easily find the ```best tuition```.
+
+By sorting the array, we can have the arrangement beginning with the smallest fee that the cow wills pay and ending with the biggest fee that the cow wills pay.
+
+We can easily calculate the amount of total money that can be generated by this array because the index of the array and the ```max tuition``` is related.
+
+For example, starting with the zero position of the array, by multiplying the number of cows that will pay a bigger amount of tuition compared to the cow at the zero position with the tuition that the cow at the zero position wills pay, we can easily know the ```best tuition``` that the cows can pay.
+
+After calculating the zero position, by moving to the next position which is the first position, again we can simply multiply the number of cows that will pay a bigger amount of tuition compared to the cow at the first position, which will be substracting -1 from the value of the previous example, with the tuition that the cow at the first position wills to pay.
+
+By comparing the first and second values that we have calculated through this example, we can know the ```best tuition``` that the cows can pay.
+
+The number of cows that will pay bigger tuition than the cow at a certain index could be easily calculated through sorting, and this is the point why sorting is crucial in this problem.
+
+By sorting it, we can know the value of the number of cows that will pay bigger tuition that the cow at a certain index by subtracting the index value from the length of the array.
+
+For example, if I want to find out how many cows are willing to pay the tuition that the cow at index 2 wants to pay, I could easily do this by sorting the array and subtracting 2 from the length of the array because all of the cows that have a bigger index than 2 will be able to pay the fee since we sorted it.
+
+By comparing these values, we can have the maximum amount of tuition that the cows can pay which is indicated as the ```best tuition```.
```

Comment:
latex numbers and try to make variable names 1 word

also the solution is kinda lengthy and convoluted, could you trim out unnecessary parts?

Reply:
Changed it